src/index.html

```diff
@@ -9,5 +9,6 @@
 </head>
 <body>
     <h1>Statistics UI</h1>
+    <input type="seach" placeholder="Search Statistics">
```

Comment:
@codewithsakshi type field should be "search", not "seach"

Reply:
Fixed it